src/material-examples/autocomplete-display/autocomplete-display-example.ts

```diff
@@ -42,8 +42,8 @@ export class AutocompleteDisplayExample {
       option.name.toLowerCase().indexOf(name.toLowerCase()) === 0);
   }
 
-  displayFn(user: User): string {
-    return user ? user.name : user;
+  displayFn(user?: User): string | undefined {
+    return user ? user.name : undefined;
```

Comment:
I'd with the following:

    displayFn(user?: User): string | undefined {
      return user ? user.name : user;
    }

... however it doesn't go well in Stackblitz (maybe because there we don't have `strictNullChecks`).